lib/cli/cli.js

```diff
@@ -59,15 +59,15 @@ exports.main = (argv = process.argv.slice(2), mochaArgs) => {
       debug('caught error sometime before command handler: %O', err);
       yargs.showHelp();
       console.error(`\n${symbols.error} ${ansi.red('ERROR:')} ${msg}`);
-      process.exitCode = 1;
+      process.exit(1);
     })
     .help('help', 'Show usage information & exit')
     .alias('help', 'h')
     .version('version', 'Show version number & exit', version)
     .alias('version', 'V')
     .wrap(process.stdout.columns ? Math.min(process.stdout.columns, 80) : 80)
     .epilog(
-      `Mocha Resources
+      `${ansi.reset("Mocha Resources")}
```

Comment:
This epilog(ue) wasn't being printed before. Now it is. On error, everything is red, so this `ansi.reset` is needed to make the colors actually show through.